docs/src/getting-started/installation.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+  :description: Tensile documentation and API reference
+  :keywords: Tensile, GEMM, Tensor, ROCm, API, Documentation
+
+.. _installation:
+
+********************************************************************
+Installation
+********************************************************************
+
+You can obtain Tensile as part of ROCm installation or choose to install Tensile from source. This document discusses both the methods to obtain Tensile.
+
+Install ROCm
+============
+
+1. Install ROCm for your platform. For installation instructions, refer to `Linux <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/quick-start.html>`_ or Windows `Windows <https://rocm.docs.amd.com/projects/install-on-windows/en/latest/index.html>` installation guide.
+
+   After the installation is complete, you can find the binaries and libraries in ``/opt/rocm``. Installing ROCm provides compilers such as ``amdclang++``, and other useful tools including ``rocminfo`` and ``rocprofv2``.
```

Comment:
@bstefanuk Is this information related to Tensile installation in any way?

Reply:
I would say we should move line 18 to the Programmer's Guide and update it to day
```suggestion
   After the installation is complete, ROCm compilers such as ``amdclang++`` can be found in ``/opt/rocm``, which is called implicitly throughout the various Tensile applications.
```

Reply:
For line 18, no, we can remove it. But please keep step number 1 for completeness.

Reply:
Done.